purchases/src/test/java/com/revenuecat/purchases/paywalls/PaywallDataTest.kt

```diff
@@ -0,0 +1,51 @@
+package com.revenuecat.purchases.paywalls
+
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import java.io.File
+import java.net.URL
+import java.util.*
+
+class PaywallDataTest {
+
+    @Test
+    fun `test PaywallData properties`() {
+        val json = File("src/test/resources/paywalldata-sample1.json").readText()
+
+        val paywall: PaywallData = Json.decodeFromString(json)
+
+        assertThat(paywall.templateName).isEqualTo("1")
+        assertThat(paywall.assetBaseURL).isEqualTo(URL("https://rc-paywalls.s3.amazonaws.com"))
+        assertThat(paywall.revision).isEqualTo(7)
+        assertThat(paywall.config.packages).containsExactly("\$rc_monthly", "\$rc_annual", "custom_package")
```

Comment:
Does this verify order too? That's super important because it determines UI order.